rules/falco_rules.yaml

```diff
@@ -1482,6 +1482,7 @@
     and not perl_running_centrifydc
     and not runuser_reading_pam
     and not user_known_read_sensitive_files_activities
+    and not (container and user_trusted_containers)
```

Comment:
Instead of using user_trusted_containers, which is kind of a catch-all list for other allowed behaviors like privileged, sensitive mounts, etc, can you define a new list just for this behavior?